tools/ctl/ctl.go

```diff
@@ -157,6 +157,8 @@ var getResults = &cobra.Command{
 			return
 		}
 		switch *format {
+		case "":
```

Comment:
you can use a comma instead: `case "", "summary"`

Reply:
Done